### ISSUE TITLE
Install on Windows fails

### ISSUE THREAD

**Issue by jorol** [gbv/PICA-Data#15]
ENV: Windows 7 32-Bit, Strawberry Perl 5.018002

``` terminal
$ cpanm PICA::Data
--> Working on PICA::Data
Fetching http://www.cpan.org/authors/id/V/VO/VOJ/PICA-Data-0.30.tar.gz ... OK
Configuring PICA-Data-0.30 ... OK
Building and testing PICA-Data-0.30 ... FAIL
! Installing PICA::Data failed. See H:\.cpanm\work\1455871673.3260\build.log for details. Retry with --force to force install it.
```

Log:

``` terminal
...
t/15-path.t ............. ok
Read more bytes than requested. Do you use an encoding-related PerlIO layer? at H:\.cpanm\work\1455871673.3260\PICA-Data-0.30\blib\lib/PICA/Parser/XML.pm line 46.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 39.
t/20-parser.t ........... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 39 subtests passed 
t/30-writer.t ........... ok
t/40-holdings.t ......... ok
t/release-pod-syntax.t .. skipped: these tests are for release candidate testing

Test Summary Report
-------------------
t/20-parser.t         (Wstat: 65280 Tests: 39 Failed: 0)
  Non-zero exit status: 255
  Parse errors: No plan found in TAP output
Files=7, Tests=102,  9 wallclock secs ( 0.17 usr +  0.13 sys =  0.30 CPU)
Result: FAIL
```

See also CPAN Testers Matrix <http://matrix.cpantesters.org/?dist=PICA-Data+0.31>

**Comment by jorol** [comment]
Seems to be a bug in test `20-parser.t`, as this test runs fine:

```perl
use strict;
use warnings;
use Test::More;
use utf8;

use PICA::Parser::XML;

my $data = do { local (@ARGV,$/) = 't/files/pica.xml'; <> };
# read from string reference
my $parser = PICA::Parser::XML->new(\$data, bless => 1 );
isa_ok $parser, "PICA::Parser::XML";
my $record = $parser->next;
isa_ok $record, 'PICA::Data';
is $record->{record}->[6]->[7], '柳经纬主编;', 'Unicode';

done_testing;
```

**Comment by jorol** [comment]
fixed in pr #17
